sql/table.cc

```diff
@@ -8479,6 +8484,199 @@ LEX_CSTRING *fk_option_name(enum_fk_option opt)
   return names + opt;
 }
 
+TR_table::TR_table(THD* _thd, bool rw) : thd(_thd)
```

Comment:
I suggest to use `read_write` instead of `rw` here.

Reply:
Don't see much reason in rename.